app/models/server/models/Subscriptions.js

```diff
@@ -1330,6 +1330,22 @@ export class Subscriptions extends Base {
 
 		return this.update(query, update, { multi: true });
 	}
+
+	setIsChatOnHold(roomId) {
+		return this.update(
+			{ rid: roomId },
+			{ $set: { isChatOnHold: true } },
+			{ multi: true },
+		);
+	}
+
+	unsetIsChatOnHold(roomId) {
+		return this.update(
+			{ rid: roomId },
+			{ $unset: { isChatOnHold: 1 } },
+			{ multi: true },
```

Comment:
Why `multi: true` if you're updating just one record?

Reply:
I've added this to handle the case where multiple agents have subscription to a room.

WDYT? should we handle this scenario?